app/assets/v2/js/pages/profile-trust.js

```diff
@@ -657,6 +657,152 @@ Vue.component('sms-verify-modal', {
   }
 });
 
+Vue.component('ens-verify-modal', {
+  data: function() {
+    return {
+      showValidation: false,
+      static_url: document.contxt.STATIC_URL,
+      validationStep: 0,
+      validationError: '',
+      validationErrorMsg: '',
+      validated: false,
+      verificationEthAddress: '',
+    };
+  },
+  mounted: function() {
+    $(document).on('click', '#verify-ens-link', function(event) {
+      event.preventDefault();
+      this.showValidation = true;
+    }.bind(this));
+    this.checkENSValidation();
```

Comment:
Why not use @click on the `#verify-ens-link` selector to make more use of vue's rich functionality.

Reply:
that was my first attempt, but `verify-ens-link` in the `tab-trust.html` doesn't use vue.